umake/frameworks/games.py

```diff
@@ -210,3 +210,42 @@ def post_install(self):
                         exec='"{}" %f'.format(self.exec_path),
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Superpowers(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Superpowers", description=_("The HTML5 2D+3D game maker"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         download_page="https://github.com/superpowers/superpowers/releases/latest",
+                         dir_to_decompress_in_tarball='superpowers*',
+                         desktop_filename="superpowers.desktop",
+                         required_files_path=["Superpowers"])
+
+    def parse_download_link(self, line, in_download):
+        """Parse Superpowers download links"""
+        url = None
+        if "-linux-" in line:
+            in_download = True
+
+        if in_download:
+            regexp = r'href="(.*-x64.zip)"'
```

Comment:
As I said, it's not needed at the moment. It would have to be changed in the others as well.
As a structure I'd propose something like:
```
if "-linux-" in line and self.bits in line:
            in_download = True
```
and then keep just one regex.

I know that having
`self.bits = '' if platform.machine() == 'i686' else 'x86_64'`
does not solve the issue, that's why I changed it in the nodejs file to something like:
```
arch_trans = {
        "amd64": "x64",
        "i386": "x86"
    }
```
`if "linux-{}.tar.xz".format(self.arch_trans[arch]) in line:`

if we need to replace parts, or simply use platform.machine().

It seems better to me to add a line in arch_trans (could be renamed to available_arch?)

If this is something that can be useful the nodejs has arm binaries, so that could be a place to start (I don't have an arm device running linux, so I can't test it, but it could be interesting)